spec/namespaces/cat.yaml

```diff
@@ -291,6 +291,9 @@ paths:
       operationId: cat.nodeattrs.0
       x-operation-group: cat.nodeattrs
       x-version-added: '1.0'
+      x-distributions-excluded:
```

Comment:
@dblock  coming from https://github.com/opensearch-project/opensearch-java/pull/1152#discussion_r1726911495, I understand the bias towards supporting AWS, but I honestly believe the spec has to be vendor neutral. I believe it should be feasible to keep the distribution specific details outside of the spec and allow the distribution specific spec to be produced if needed

Reply:
I think you can/should open an issue copy-pasting the above about removing x-distribution-excluded.

When I introduced this feature I did not intend it to be biased towards supporting AWS in any way, but I did want to run the tests against Amazon Managed so I annotated some APIs. We would welcome other keywords here, be it `oracle-oci`, `aiven-managed` or something platform-specific like `community-freebsd` in the short term. Therefore I believe the spec *is* vendor netural today. I *think* it's ok to mention vendors in the spec this way, but we can talk about it. 

What you are looking at is the source for the spec, not the spec. We currently publish the all-versions spec in a [release](https://github.com/opensearch-project/opensearch-api-specification/releases). This repo is capable of producing a version-specific spec (e.g. 1.3.x, without APIs added in/after 2.0 for example or vice-versa) (see https://github.com/opensearch-project/opensearch-api-specification/blob/main/DEVELOPER_GUIDE.md#spec-merger), and that could be extended to produce a vendor-specific spec like (e.g. `-amazon-managed.yaml`).